### DISCUSSION TITLE
Does the ability to use keyboard shortcuts to switch between tiled windows exist

### DISCUSSION THREAD

**Discussion by robertstrom** [rxhanson/Rectangle#1254] [Q&A]
Hello,

I just found Rectangle and either it, or Rectangle Pro appear to be what I am looking for but would also like to be able to easily switch between windows using keyboard shortcuts (not CMD + TAB) such as a modifier key(s) and an arrow key.

Does this ability exist and I just have not found it yet? If not, are there any plans to implement this ability?

Thanks!

**Comment by rxhanson** [comment]
The built-in shortcut for switching between windows is cmd+\`

I have long had a backlog item for switching between windows of any app that are tiled, but haven't made it to it, and it's not in Rectangle Pro at this time either. I see no reason to turn a good pull request with this functionality.
